celeste/src/login/login_util.rs

```diff
@@ -88,14 +88,8 @@ pub fn username_input() -> EntryRow {
     let input = EntryRow::builder().title("Username").build();
     input.connect_changed(|input| {
         let _text = input.text();
-
-        if input.text().contains(':') {
-            input.add_css_class("error");
-            input.set_tooltip_text(Some("Username isn't allowed to contain ':'."));
-        } else {
-            input.remove_css_class("error");
-            input.set_tooltip_text(None);
-        }
+        input.remove_css_class("error");
+        input.set_tooltip_text(None);
```

Comment:
Since we remove all calls to `input.add_css_class("error")`, we can also go ahead and remove all the `input.remove_css_class("error")` calls. The same logic applies for the `input.set_tooltip_text` calls.